testproject/Assets/Tests/Manual/SceneTransitioningAdditive/NetworkManagerMonitor.cs

```diff
@@ -0,0 +1,22 @@
+using UnityEngine;
+using Unity.Netcode;
+
```

Comment:
This component can be added to the same GameObject the NetworkManager component is assigned to in order to prevent multiple NetworkManager instances from being instantiated if the same scene is loaded.